src/app/features/course-info/course-info.component.html

```diff
@@ -1,17 +1,15 @@
-<!-- Replace ::text:: with your code -->
-
-<div class="title">::Title::</div>
+<div class="title">{{course.title}}</div>
 <div class="course-info">
   <div class="course-info__description">
-    ::Description::
+    {{course.description}}
   </div>
   <div class="course-info__info">
-    <div><b>ID: </b><span class="course-info__info-id">::id::</span></div>
-    <div><b>Duration: </b><span class="course-info__info-duration">::duration::</span></div>
-    <div><b>Created: </b><span class="course-info__info-date">::date::</span></div>
-    <div><b>Authors: </b><span class="course-info__info-authors">::authors::</span></div>
+    <div><b>ID: </b><span class="course-info__info-id">{{course.id}}</span></div>
+    <div><b>Duration: </b><span class="course-info__info-duration">{{course.duration}}</span></div>
+    <div><b>Created: </b><span class="course-info__info-date">{{course.creationDate}}</span></div>
+    <div><b>Authors: </b><span class="course-info__info-authors">{{course.authors}}</span></div>
   </div>
 </div>
 <div class="navigation-block">
-  <!--Enter your code here-->
+  <app-button buttonText="Back"></app-button>
```

Comment:
Update the component input notation please

Reply:
As an extra, can you think of a solution to pass the actual text to the button in a clean manner?
Let's just assume that we have quite a few buttons with different labels. Each label will occur multiple times. On a sunny monday morning, the  product owner ask to change all 'Back' label to 'Back to previous'.
We want to make the change only one, not for each use case. Please introduce a solution to keep track of all possible button label texts and use it across the application.